src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/model/entity/session/PermittedSessionLocation.kt

```diff
@@ -33,9 +32,6 @@ data class PermittedSessionLocation(
   @JoinColumn(name = "PRISON_ID", updatable = false, insertable = false)
   val prison: Prison,
 
-  @Column(name = "TYPE", nullable = false)
-  val type: PermittedType,
-
   @ManyToMany(mappedBy = "permittedSessionLocations")
   var sessionTemplates: MutableSet<SessionTemplate> = mutableSetOf(),
```

Comment:
We could make this var private so as not to have a getter as we only want PermittedSessionLocations as part of SessionTemplate. But not sure if it will impact how JPA works.

Reply:
removed this now but rich reckons we should give it a go

Reply:
Should I put this back in then? I have the script done for this also... so if we do that it's done